src/main/java/ServerPort.java

```diff
@@ -0,0 +1,37 @@
+package com.td.HTTPServer;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.Date;
+
+public class ServerPort {
```

Comment:
When I see `serverPort` I think of the port number. I think you could find a more descriptive name for this class.